javascript/javascript2/week1/hyfBya/main.js

```diff
@@ -0,0 +1,25 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const pUl = document.createElement("ul");
+pUl.classList.add("products-list");
+document.querySelector("main").appendChild(pUl);
+
+function renderProducts(products) {
+  for (let i = 0; i < products.length; i++) {
+    let product = document.createElement("li");
+    product.innerHTML =
```

Comment:
Great use of `innerHtml`! 
 
You might be interested in knowing that you can make life a little easier with string interpolation, like: 
```js
product.innerHTML = `
        <h3>Title: ${products[i].name}</h3>
        <p>Price: ${products[i].price}</p>
        <p>Rating: ${product[i].rating}</p>
        `;
```